script/upgrades/MU09/MU09Checks.sol

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8;
+pragma experimental ABIEncoderV2;
+
+import { GovernanceScript } from "script/utils/mento/Script.sol";
+import { console } from "forge-std/console.sol";
+import { Contracts } from "script/utils/mento/Contracts.sol";
+import { Test } from "forge-std/Test.sol";
+
+import { IGovernanceFactory } from "script/interfaces/IGovernanceFactory.sol";
+import { ITransparentUpgradeableProxy } from "mento-core-2.6.0-tp/TransparentUpgradeableProxy.sol";
+import { ProxyAdmin } from "mento-core-2.6.0-tp/ProxyAdmin.sol";
```

Comment:
nit: we use ProxyAdmin in checks but in MU09 we use IProxyAdminLite

Reply:
Won't change unless you think it's necessary. I switched to using the full implementation in the checks when adding the upgrade check